src/tests/frontend/runner.css

```diff
@@ -122,19 +114,6 @@ body {
   font-size: 9px;
   margin-left: 5px;
   padding: 2px 5px;
-  color: white;
-  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.2);
-  -moz-box-shadow: inset 0 1px 1px rgba(0,0,0,.2);
-  box-shadow: inset 0 1px 1px rgba(0,0,0,.2);
-  -webkit-border-radius: 5px;
-  -moz-border-radius: 5px;
-  -ms-border-radius: 5px;
-  -o-border-radius: 5px;
-  border-radius: 5px;
-}
-
-#mocha .test.pass.fast .duration {
-  display: none;
 }
```

Comment:
I prefer the colored box around the duration. Maybe just delete this `display: none` and leave the rest alone?

Reply:
I made it prettier.

![image](https://user-images.githubusercontent.com/220864/107871627-3601b580-6e9b-11eb-9174-501a35296ccb.png)

Reply:
It's 2021.  Drop shadows are ashes.